CMakeLists.txt

```diff
@@ -0,0 +1,177 @@
+cmake_minimum_required(VERSION 3.1)
+cmake_policy(SET CMP0074 NEW)
+
+include(${CMAKE_CURRENT_LIST_DIR}/cmake/Hunter/HunterGate.cmake)
+HunterGate(
+    URL "https://github.com/ruslo/hunter/archive/v0.23.179.tar.gz"
+    SHA1 "21b2f139feb9f3c37e367750013aee15e1c45355"
+)
+
+project(imgui VERSION 1.70)
+set(IMGUI_VERSION "1.70")
```

Comment:
`imgui_VERSION` will be introduced by `project` command:
- https://cmake.org/cmake/help/latest/command/project.html#options

No need to introduce a custom variable.

Reply:
thought I had removed that, oops